idl/stm32xx-sys.idol

```diff
@@ -95,5 +95,35 @@ Interface(
             reply: Simple("[u32; 3]"),
             idempotent: true,
         ),
+
+        // Configures some set of EXTI sources associated with the caller,
+        // using the caller's notification bit space to name them. Any
+        // sources included in the `mask` will be affected.
+        //
+        // If neither rising nor falling is true, the pin will not be
+        // capable of producing an interrupt, even if enabled.
```

Comment:
is this case something we ought to check for and reply fault?

Reply:
I feel like that might be too aggressive, I am not convinced that it isn't a potentially useful technique.

Reply:
my thinking is that we have a separate interface for saying "hey, i don't want to receive this interrupt right now" (in `gpio_irq_control`), so the code that calls `gpio_irq_configure` and asks for neither the rising nor the falling edge probably didn't *intend* to do that? 

but, maybe there are occasions where it feels more correct to call this rather than `gpio_irq_configure` to disable an IRQ — perhaps other code will call `gpio_irq_control` to enable an IRQ unconditionally, and this is useful as an override? i'm fine with permitting this if you think there are valid uses for it. 🤷‍♀️

Reply:
I guess it's easier to permit it later if it's useful, than it is to ban it later if it isn't. I'll add it as an error case with a note about how to remove it if desired.